src/phpdoc/parser.go

```diff
@@ -58,22 +64,47 @@ func IsPHPDocToken(t *token.Token) bool {
 	}
 
 	if t.ID == token.T_COMMENT {
-		return bytes.HasPrefix(t.Value, []byte("/* @var "))
+		return IsPossiblePhpDoc(t.Value)
 	}
 
 	return true
 }
 
+// IsPossiblePhpDoc checks if /* */ comments contain annotations, which may mean that
+// it is phpdoc, but there is a mistake when there is one asterisk instead of two at
+// the beginning of a comment.
+func IsPossiblePhpDoc(value []byte) bool {
```

Comment:
Does phpstorm understand any kind of a single-line phpdoc with `/* @tag */` syntax?
If it does, we could check for any tag, not for a tag from a list.
And rename a function to `ContainsTag` (again, mentioning phpdoc in a name is not always needed if this function is a part of phpdoc package).

Reply:
Done.